src/main/java/com/backbase/ct/bbfuel/configurator/PaymentsConfigurator.java

```diff
@@ -49,13 +50,15 @@ public void ingestPaymentOrders(String externalUserId) {
             .getUsDomesticWireArrangements();
         List<ArrangementsByBusinessFunctionGetResponseBody> achDebitArrangements = productSummaryPresentationRestClient
             .getAchDebitArrangements();
+        List<ArrangementsByBusinessFunctionGetResponseBody> usForeignWireArrangements = productSummaryPresentationRestClient
+            .getUSForeignWireArrangements();
 
         int randomAmount = CommonHelpers
             .generateRandomNumberInRange(globalProperties.getInt(CommonConstants.PROPERTY_PAYMENTS_MIN),
                 globalProperties.getInt(CommonConstants.PROPERTY_PAYMENTS_MAX));
 
         if (sepaCtArrangements.size() != 0 && usDomesticWireArrangements.size() != 0
-            && achDebitArrangements.size() != 0) {
+            && achDebitArrangements.size() != 0 && usForeignWireArrangements.size() !=0) {
```

Comment:
Needs formatting

Reply:
please use `!isEmpty(...)` instead of `size() != 0`

Reply:
done